status-im/status-desktop
# Add new member after typing message and pressing enter

### What does the PR do

Fixes https://github.com/status-im/status-desktop/issues/8349

When the member selector menu is open, sending a message adds/removes the members and then sends the message. Therefore, the new member will also receive the message.

The first commit is just a cleanup of code taht is never used

### Affected areas

Group chat member selection (for existing chat)

### Architecture compliance

My PR is consistent with this document: [Status Desktop Architecture Guide](https://github.com/status-im/status-desktop/blob/master/CONTRIBUTING.md)

### Screenshot of functionality (including design for comparison)

[add-remove-members.webm](https://github.com/user-attachments/assets/028a67e4-fd02-4dcf-9d6a-3c8f2544779a)

### Impact on end user

Enables the user to add or remove members more easily

### How to test

0. Have a group channel
1. Open the "add/remove members" menu
2. add a member (don't press Confirm)
3. Enter a message
4. Press Enter
5. See that the member is added and they received the message
6. Reopen the menu
7. Remove the user (don't press confirm again)
8. Send another message
9. The user is removed and they don't receive the message

Also, using the Confirm as before works as usual

### Risk 

Tick **one**:

Worst case the new feature doesn't work
